transport_nantes/stripe_app/models.py

```diff
@@ -87,13 +88,21 @@ class Donation(models.Model):
                                               default=None, null=True)
 
     def __str__(self):
-        if self.periodicity_months == 0:
-            return self.first_name + " " + self.last_name + \
-                " ({}) ".format(self.email) +\
-                str(round(float(self.amount_centimes_euros/100), 2)) \
-                + "€ ONE-TIME" + str(self.timestamp)[:19]
+
+        display_name = ""
+        amount_given = str(
+            round(float(self.amount_centimes_euros/100), 2)) + "€"
```

Comment:
Please force yourself to think separation of view (function) and display.  The euro sign is not part of the amount.  The rounding should be in the template (django floatformat filter).

Reply:
In what template ? This is purely for admin panel.

Reply:
ah, ok, I have less intuition about what the right way to interact with the admin panel is.

i missed that this was `__string__()`.